### PR TITLE
Link to project page.

### PR DESCRIPTION
**Pull request by jhedstrom**
This will link module's to their d.o. project page (this doesn't make sense though for projects not currently hosted there though...).

### CODE REVIEW

**Comment by Berdir** [comment]
Was thinking about this too. Can we move the link out of twig and into a variable that's set in PHP. I think we'll end up adding more information to the table, and some of it might be controlled manually. So I had the idea to start a .yaml file with configurations, which could be used to set the url to a project if we can't get it automatically

**Comment by jhedstrom** [comment]
How's this look? I only overrode the redirect url in `config.yml`, but could override others if this looks like the right direction.

**Comment by Berdir** [comment]
Looks good to me! :) I would have been OK with just hardcoding the url in PHP, the other part was just explaining the direction in which I want to go, but starting with that works for me as well ;)

File: parse_results.php [hunk 1]
```diff
@@ -2,6 +2,15 @@
 
 require_once 'vendor/autoload.php';
 
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Drupal.org project url pattern.
+ */
+const DRUPAL_PROJECT_URL='https://www.drupal.org/project';
```
Comment (Berdir) [inline parse_results.php]: minor coding standard issue here, missing spaces.

**Comment by jhedstrom** [comment]
Coding standards fixed.

**Pull request merged**